internal/install/recipe_installer.go

```diff
@@ -408,7 +408,6 @@ func (i *RecipeInstaller) userAcceptsInstall(r types.Recipe) bool {
 }
 
 func (i *RecipeInstaller) fail(err error) error {
-	log.Error(err)
```

Comment:
Good call, we are logging this one a level up in the stack as well.  Do we want to get rid of `fail()` altogether and just call `i.reportComplete()` in its place maybe?